mappings.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0"?>
+<mappings>
+  <mapping>
+    <path>~/navigation/position/latitude</path>
+    <sentence>
+      <id>RMC</id>
+      <fieldset type="coordinate">
+        <field>2</field>
+        <field>3</field>
+      </fieldset>
+      <filter type="if">
+        <field>1</field>
+        <value transform="uppercase">A</value>
+      </filter>
+    </sentence>
+  </mapping>
+
+  <mapping>
+    <path>~/navigation/position/longitude</path>
```

Comment:
Since position.latitude, position.longitude and position.timestamp are not related how do you propose to create just one delta message out of these, or does the code already handle this?



This demonstrates the fact that I find the association order of this mapping a bit cumbersome - the mapping would be much more natural if it were from input (sentence) to output (delta messages eg. Signal K paths).

See also https://github.com/SignalK/n2k-signalk/blob/master/n2kMappings.js (hmm `node` should be renamed to `path`).